src/main/java/mate/academy/connectionutil/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.connectionutil;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/bookstore";
+    private static final Properties DB_PROPERTIES;
+    private static final String USER_NAME = "root";
+    private static final String PASSWORD = "Kate5819";
```

Comment:
Hardcoding database credentials (username and password) in the source code is a security risk. Consider using environment variables or a secure vault to store sensitive information.